api/src/app.ts

```diff
@@ -1,62 +1,34 @@
-import express, { Express } from 'express';
+import express from 'express';
 import { PrismaClient } from '@prisma/client';
 import morgan from 'morgan';
-import helmet from 'helmet';
-import cors from 'cors';
-import swaggerUi from 'swagger-ui-express';
-import { ValidateError } from 'tsoa';
-import { RegisterRoutes } from '../tsoa/routes';
+import helmet from 'helmet'
+import * as swaggerUi from 'swagger-ui-express';
+import Router from './routes/route';
 
-const app: Express = express();
-
-/************************************************************************************
- *                              Basic Express Middlewares
- ***********************************************************************************/
+const app = express();
 
 app.set('json spaces', 4);
 app.use(express.json());
+app.use(morgan("tiny"));
+app.use(express.static("public"));
 app.use(express.urlencoded({ extended: true }));
+app.use(helmet());
 
-// Handle logs in console during development
-if (process.env.NODE_ENV === 'development' || process.env.NODE_ENV === 'development') {
-  app.use(morgan('dev'));
-  app.use(cors());
-}
-
-// Handle security and origin in production
-if (process.env.NODE_ENV === 'production' || process.env.NODE_ENV === 'production') {
-  app.use(helmet());
-}
```

Comment:
Is this functionality replaced anywhere? Or just deleted?

Reply:
I decided to set the Helmet-Header also in our Development Environment. Further I switched the morgan dev->tiny. Relating to cors() I was unsure, why I should enable it??

Reply:
Alright, yeah I think the cors headers are just meant as a convenience for dev, but as long as we have morgan logging somewhere I think it's fine.